### ISSUE TITLE
[BUG] Kruise manager 0.10.0 restarts without workloadspread CRD

### ISSUE THREAD

**Issue by dragondscv** [openkruise/kruise#793]
**What happened**:
Kruise-manager starts but crashes after a while because workloadspread CRD does not exist:
```
I1105 16:08:10.891160       1 controller.go:142] controller "msg"="Starting EventSource" "controller"="workloadspread-controller" "source"={"Type":{"metadata"
:{"creationTimestamp":null},"spec":{"targetRef":null,"subsets":null,"scheduleStrategy":{}},"status":{}}}
E1105 16:08:11.226173       1 source.go:117] controller-runtime/source "msg"="if kind is a CRD, it should be installed before calling Start" "error"="no match
es for kind \"WorkloadSpread\" in version \"apps.kruise.io/v1alpha1\""  "kind"={"Group":"apps.kruise.io","Kind":"WorkloadSpread"}
E1105 16:08:11.726513       1 main.go:178] setup "msg"="problem running manager" "error"="no matches for kind \"WorkloadSpread\" in version \"apps.kruise.io/v
1alpha1\""
```
By default workloadspread should be disabled in feature gate but kruise manager is still looking for it in 0.10.0. Looks like workloadspread controller does not check group version kind as other controllers do: https://github.com/openkruise/kruise/blob/1cc55e41066986e275194710ff59f1b4160d4a7a/pkg/controller/advancedcronjob/advancedcronjob_controller.go#L58
Other controllers are working fine without corresponding CRDs.

**What you expected to happen**:
Kruise manager should not crash without workloadspread CRD.

**How to reproduce it (as minimally and precisely as possible)**:
Deploy kruise manager 0.10.0 without installing workloadspread CRD.

**Anything else we need to know?**:

**Environment**:
- Kruise version: 0.10.0
- Kubernetes version (use `kubectl version`): 1.17
- OS (e.g: `cat /etc/os-release`): CentOS 7.8
- Kernel (e.g. `uname -a`): 4.19.128
- Install tools: helm
- Others:

**Comment by wgitg** [comment]
Same issue. We want to upgrade to 0.9.0, not the latest version, but the controller manager crashes due to the lack of `WorkloadSpread` CRD installed

**Comment by veophi** [comment]
I guess you upgrade your `kruise-manager` but do not upgrade `kruise CRDs`.

Cloud you describe your installation commands?

**Comment by dragondscv** [comment]
Yes, we didn't install workloadspread CRD. But I think even without the crd, the workloadspread controller should still work(like other controllers, we didn't install other CRDs as well)?

**Comment by veophi** [comment]
> Yes, we didn't install workloadspread CRD. But I think even without the crd, the workloadspread controller should still work(like other controllers, we didn't install other CRDs as well)?

Oh, I see.  Sorry....  it's indeed my fault.

**Comment by dragondscv** [comment]
Thanks for fixing this!

**Comment by dragondscv** [comment]
Just curious, what is the release process? Will it be included in the next patch version of 0.10?

**Comment by FillZpp** [comment]
> Just curious, what is the release process? Will it be included in the next patch version of 0.10?

Today we have released `v0.10.1` and `v1.0.0-alpha.2` that all contain the fixing code. Feel free to use them. https://github.com/openkruise/kruise/releases
